server/service_test.go

```diff
@@ -537,199 +539,22 @@ func TestGetHeaderBids(t *testing.T) {
 		require.Equal(t, 1, backend.relays[0].GetRequestCount(path))
 
 		// Value should be 12345 (min bid is 12345)
-		resp := new(types.GetHeaderResponse)
```

Comment:
duplicate code originally used to test capella. Now that bellatrix get header is removed, we can remove these tests